packages/core-utils/src/common/metrics.ts

```diff
@@ -0,0 +1,31 @@
+import prometheus, {
+  collectDefaultMetrics,
+  DefaultMetricsCollectorConfiguration,
+  Registry,
+} from 'prom-client'
+
+export interface MetricsOptions {
+  prefix: string
+  labels?: Object
+}
```

Comment:
@snario i based `metrics.ts` off of `logger.ts` and [The Graph's Metrics](https://github.com/graphprotocol/common-ts/blob/master/packages/common-ts/src/metrics/index.ts). decided to keep it simple and not include passing in another registry as an option; lmk if you think that might be necessary for us though!